### PR TITLE
add enter67_pcb

### PR DESCRIPTION
**Pull request by LXF-YZP**
## Description

Add a 67 layout pcb

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/enter67/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_PGDN,
+    KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    MO(1),   KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT),
+
+[1]=LAYOUT(
+    KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                   KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS)
+
```
Comment (dunk2k) [inline keyboards/enter67/keymaps/via/keymap.c:34]:
```suggestion
[0] = LAYOUT_65_ansi_blocker(
    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_END,
    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGUP,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_PGDN,
    KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    MO(1),   KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT),
[1] = LAYOUT_65_ansi_blocker(
    KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                   KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS),
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/enter67/info.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+              {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+              {"label": "K91", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
+              {"label": "K92", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
+              {"label": "Left", "matrix": [4, 12], "x": 13, "y": 4, "w": 1},
+              {"label": "Down", "matrix": [4, 13], "x": 14, "y": 4, "w": 1},
+              {"label": "Right", "matrix": [4, 14], "x": 15, "y": 4, "w": 1}
+          ]
+      }
+  }
+}
```
[the remark this answers is not in the document]
Comment (LXF-YZP) [reply keyboards/enter67/info.json:105]: I executed the command "qmk format-json info.json " ，Please accept my heartfelt thanks.

File: keyboards/enter67/config.h
Comment (waffle87) [inline keyboards/enter67/config.h:1]: This file should be removed.

File: keyboards/enter67/halconf.h
Comment (waffle87) [inline keyboards/enter67/halconf.h:1]: This file should be removed.

File: keyboards/enter67/mcuconf.h
Comment (waffle87) [inline keyboards/enter67/mcuconf.h:1]: This file should be removed.

Comment (LXF-YZP) [reply keyboards/enter67/mcuconf.h:1]: thank you

Comment (LXF-YZP) [reply keyboards/enter67/mcuconf.h:1]: I had no problem compiling the firmware locally, but a check failed after I submitted it. I'm trying to figure it out

File: keyboards/enter67/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# ENTER67 - PCB
+
+![enter67](https://imgur.com/igF8Lurh.jpg)
+
+A 67 layout pcb.
```
Comment (dunk2k) [inline keyboards/enter67/readme.md:5]:
```suggestion
67 layout PCB, with option for stepped Caps Lock
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/enter67/info.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    "matrix_pins": {
+        "cols": ["A7", "B0", "B1", "B2", "B10", "B11", "B12", "B14", "A9", "A10", "A8", "B6", "B7", "B5", "B4"],
+        "rows": ["A1", "A2", "A3", "A4", "A5"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0xAA66",
+        "vid": "0x1000"
```
[the remark this answers is not in the document]
Comment (LXF-YZP) [reply keyboards/enter67/info.json:26]: I have no right to use it, I will modify vid, thank you

**Review by waffle87 — APPROVED** [review]

File: keyboards/enter67/mcuconf.h
Comment (LXF-YZP) [reply keyboards/enter67/mcuconf.h:1]: Please accept my heartfelt thanks @lesshonor

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**